packages/shared/src/components/icons/Bluesky/mono.svg

```diff
@@ -0,0 +1,12 @@
+<svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="24"
+    height="24"
+    fill="none"
+    viewBox="0 0 24 24"
+  >
+    <path
+      fill="currentColor"
```

Comment:
Since we have `currentColor` here we could just use this svg both for mono/color version. What do you think?

Reply:
Thanks, @ilasw and @omBratteng. Pushed https://github.com/dailydotdev/apps/pull/4052/commits/06286396aa7755e0a145347eae6b05326206c582.